config.json

```diff
@@ -659,6 +659,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 5
+      },
+      {
+        "slug": "knapsack",
+        "name": "Knapsack",
+        "uuid": "41029de7-58a6-431b-9f82-6d4603604fb3",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
```

Comment:
I think the difficulty should be bumped. Racket marks this an easy exercise (3), Common Lisp as medium (7) and Scheme as hard (8).

Reply:
Corrected.